.github/workflows/build.yml

```
@@ -0,0 +1,51 @@
+name: Create image
+
+on:
+  push:
+    branches:
+      - dev
+      - main
+      - master
```

```suggestion
      - master
      - DSR-*
```

I want this for all repos forever.